pom.xml

```diff
@@ -11,7 +11,7 @@
     <version>1.0.11-SNAPSHOT</version>
     <packaging>hpi</packaging>
     <properties>
-        <jenkins.version>2.7.3</jenkins.version>
+        <jenkins.version>2.107.1</jenkins.version>
```

Comment:
This isn't really related to the changes in this PR. It should be pulled out into its own PR.

Reply:
Removed